### PR TITLE
Update documentation for myStrom config flow

### PR DESCRIPTION
**Pull request by pail23**
## Proposed change

Update documentation for myStrom config flow

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#74719
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by c0ffeeca7 — DISMISSED** [review]
LGTM. Thank you, @pail23 :+1:

File: source/_integrations/mystrom.markdown [hunk 1]
```diff
@@ -17,43 +18,23 @@ ha_platforms:
...
+  - [Setup of myStrom Bulbs](#setup-of-mystrom-bulbs)
 - [Binary Sensor](#binary-sensor)
   - [Setup of myStrom Buttons](#setup-of-mystrom-buttons)
 - [Switch](#switch)
   - [Setup](#setup)
 
 ## Light
 
-To use your myStrom WiFi Bulb in your installation, add the following to your `configuration.yaml` file:
+The `mystrom` light platform allows you to control your [myStrom](https://mystrom.ch/) WiFi Bulbs.
```
Comment (frenck) [inline source/_integrations/mystrom.markdown]: This is now no longer a platform integration. We don't need to backtick `mystrom` anymore in this case.

File: source/_integrations/mystrom.markdown [hunk 2]
```diff
@@ -17,43 +18,23 @@ ha_platforms:
...
 
-```yaml
-# Example configuration.yaml entry
-light:
-  - platform: mystrom
-    host: IP_ADDRESS
-    mac: MAC_ADDRESS
-```
+### Setup of myStrom Bulbs
```
Comment (frenck) [inline source/_integrations/mystrom.markdown]: The configuration header is already provided by the inclusion snippet you included below.

File: source/_integrations/mystrom.markdown [hunk 3]
```diff
@@ -167,23 +148,5 @@ Make sure that you have enabled the REST API under **Advanced** in the web front
...
 
-To use your myStrom switch in your installation, add the following to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-switch:
-  - platform: mystrom
-    host: IP_ADRRESS
-```
+{%include integrations/config_flow.md %}
```
Comment (frenck) [inline source/_integrations/mystrom.markdown]: This setup instructions should only appear once for this integration?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/mystrom.markdown (hunk 1 above)
Comment (pail23) [reply source/_integrations/mystrom.markdown]: I merged the light and the switch description and removed the backticks

File: source/_integrations/mystrom.markdown (hunk 2 above)
Comment (pail23) [reply source/_integrations/mystrom.markdown]: removed

File: source/_integrations/mystrom.markdown (hunk 3 above)
Comment (pail23) [reply source/_integrations/mystrom.markdown]: merged with the light description

**Review by frenck — APPROVED** [review]
Thanks, @pail23 👍 

../Frenck

**Pull request merged**